### DISCUSSION TITLE
Can't update comfyui

### DISCUSSION THREAD

**Discussion by lycos2019** [LykosAI/StabilityMatrix#1062] [Q&A]
![asdfasdf](https://github.com/user-attachments/assets/63ad3ae9-cf04-4ca7-a3d4-5217a3e6fcc5)

trying to update 3.9ver from 3.7 but install kept failing

**Comment by mohnjiles** [comment]
If you check the Downloads tab in the bottom left, there may be some more details in there about exactly what failed. You can click the arrow button to bring the dialog box back up & it should have an error message in it. Could you post the error as well please? Thanks!
